### PR TITLE
[14.0][MIG] pos_payment_method_cashdro: Migration to version 14.0

### PR DESCRIPTION
**Pull request by chienandalu**
- [x] The PoS has been revamped from this version. We can use many features from the core payment terminal gears to simplify the original mechanics of the module.

cc @Tecnativa TT38070

ping @CarlosRoca13 @victoralmau

### CODE REVIEW

File: pos_payment_method_cashdro/models/pos_payment_method.py [hunk 1]
```diff
@@ -3,12 +3,14 @@
 from odoo import fields, models
 
 
-class AccountJournal(models.Model):
-    _inherit = "account.journal"
+class PosPaymentMethod(models.Model):
+    _inherit = "pos.payment.method"
     # TODO: In 13.0 this fields should be moved to `pos.payment.method`
     # and be addod to the payment terminal selection
```
Comment (victoralmau) [inline pos_payment_method_cashdro/models/pos_payment_method.py]: Can you delete this comment now that the change is done?

**Review by victoralmau — APPROVED** [review]
Code review OK

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu please check pre commit

**Comment by pedrobaeza** [comment]
@chienandalu please finish this.

**Pull request merged**